content/logs.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Logs
+order: 32
+description: How to use your logs
+---
+
+The logs should be your first stop when troubleshooting any Galaxy issue. When working with Galaxy support, an agent will usually begin by consulting your logs. If a clearly noted issue has been marked there, you can save time by reading the logs and iterating on your code until the error goes away.
+
+You can reach the logs dashboard by clicking on your app, then clicking the "Logs" tab.
+
+<img src="/images/logs_section.png" style="margin: 1em 0;"/>
+
+<h3 id="tabs">Tabs</h3>
+
+There are several logging tabs you can click, below the "Logs" tab itself. These are:
+
+- All: for the combined view.
+- App: for application-specific messages and errors.
+- Service: for messages generated during the build process.
```

Comment:
Weirdly this combines both "build process" and "messages saying that a container is starting or stopping"